Code/Editor/Core/LevelEditorMenuHandler.cpp

```diff
@@ -1058,7 +1058,7 @@ void LevelEditorMenuHandler::UpdateMRUFiles()
                 // Check file is still available
                 if (mruList->GetSize() > i)
                 {
-                    cryEdit->OpenDocumentFile((*mruList)[i].toUtf8().data());
+                    cryEdit->OpenDocumentFile((*mruList)[i].toUtf8().data(), true, COpenSameLevelOptions::ReopenLevelIfSame);
```

Comment:
suggestion: Would it be possible to please add a parameter comment for the `true` variable so it's clearer at the callsite to see what it's referring to (e.g. `, /*promptChanges=*/ true`)